recordedit/form.controller.js

```diff
@@ -505,7 +505,7 @@
             var transformedRow = transformRowValues(modelRow);
             columns.forEach(function (column) {
                 // If the column is a pseudo column, it needs to get the originating columns name for data submission
-                if (column.isPseudo) {
+                if (column.isPseudo && column.isForeignKey) {
```

Comment:
This case is meant for only foreign keys. Checking for column.isPseudo is redundant. Could you remove it?